processor/src/trace/mod.rs

```diff
@@ -143,6 +148,11 @@ impl ExecutionTrace {
         self.main_trace.num_rows()
     }
 
+    /// Returns a summary of the lengths of main, range and chiplet traces.
+    pub fn get_len_summary(&self) -> &TraceLenSummary {
+        &self.trace_len_summary
+    }
```

Comment:
I would just call it `trace_len_summary()` - we use `get_` prefix usually when there is some work involved in creating the returned value. Here we are just returning a reference.